src/nautilus_librarian/_helper.py

```diff
@@ -0,0 +1,46 @@
+"""Get Version from pyproject.toml"""
+
+from pathlib import Path
+from typing import Any
+
+import atoml
+from atoml import TOMLDocument
+
+from _helpers import get_unicode_document, path_to_pyproject_toml
+
+
+def get_toml_document(string: str) -> TOMLDocument:
+    """Get the Toml Document from Path"""
+
+    return atoml.parse(string=string)
+
+
+def get_version_from_toml(doc: TOMLDocument) -> str:
```

Comment:
maybe rename this function to: `extract_version_from_toml_doc` for additional clarity.